environment.yml

```diff
@@ -9,8 +9,13 @@ dependencies:
   - _libgcc_mutex=0.1
   - _openmp_mutex=4.5
   - _r-mutex=1.0.1
+  - aioeasywebdav=2.4.0
```

Comment:
Are all of these packages that your code actually imports? I try to limit my `environment.yml` files to packages I'm directly importing in my Python code (or running from the command line like `snakemake` or `pytest`) to minimize clutter. My gut feeling is that this will improve portability, since fewer explicit requirements = more wiggle room to install packages/package versions that are specific to the user's system.

That's just my opinion though, I don't feel too strongly about this. It also means you have to remember to add things to this file by hand when you add new imports, but your tests/CI should break when this is out of sync so it shouldn't be a huge concern.

Reply:
I prefer to include all the libraries (without builds) because it's automatic and more explicit. That being said, I imagine it's possible I'll run into an uninstallable environment file some day and change my opinion entirely